generator/src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java

```diff
@@ -239,49 +120,58 @@ TypeDef getType(
         if (!format.isEmpty() ) {
             switch (format) {
             case "uint64":
-                return new TypeDef("java.math.BigInteger", type, propName, goName, desc, isRequired(prop));
+                return new TypeDef("java.math.BigInteger", type, "", propName, goName, desc, isRequired(prop));
             case "RFC3339 String":
-                addImport(imports, "java.util.Date");
-                addImport(imports, "com.algorand.algosdk.v2.client.common.Utils");
-                return new TypeDef("Date", "time", propName, goName, desc, isRequired(prop));
+                return new TypeDef("Date", "time", "", propName, goName, desc, isRequired(prop));
             case "Address":
-                return getAddress(propName, goName, imports, forModel, desc, isRequired(prop));
+                return new TypeDef("Address", "address", "getterSetter", propName, 
+                        goName, desc, isRequired(prop));
+
             case "SignedTransaction":
-                addImport(imports, "com.algorand.algosdk.transaction.SignedTransaction");
-                return new TypeDef("SignedTransaction", format, propName, goName, desc, isRequired(prop));
+                return new TypeDef("SignedTransaction", format, "", propName, goName, desc, isRequired(prop));
             case "binary":
             case "byte":
             case "base64":
             case "digest":
                 if (type.contentEquals("array")) {
                     type = prop.get("items").get("type").asText(); 
-                    return getBase64EncodedArray(propName, goName, type, imports, forModel, desc, isRequired(prop));
+                    if (forModel == false) {
+                        throw new RuntimeException("array of byte[] cannot yet be used in a path or path query.");
+                    }
+                    
+                    if (format.equals("byte")) {
+                        type = format;
+                    }
+
+                    // getterSetter typeName is only used in path.
+                    return new TypeDef("", type, "getterSetter,array", 
+                            propName, goName, desc, isRequired(prop));
                 } else {
-                    return getBase64Encoded(propName, goName, type, imports, forModel, desc, isRequired(prop));
+                    return new TypeDef("byte[]", "binary", "getterSetter", 
+                            propName, goName, desc, isRequired(prop));
                 }
             case "AccountID":
                 break;
             case "BlockCertificate":
             case "BlockHeader":
-                addImport(imports, "java.util.HashMap");
-                return new TypeDef("HashMap<String,Object>", type, propName, goName, desc, isRequired(prop));
+                return new TypeDef("HashMap<String,Object>", type, "", propName, goName, desc, isRequired(prop));
```

Comment:
Yes, it is left so. It is okay for now. It will be replaced with something more or less with something similar, which does not have the term "java" in it. Something like InterpretedType.